docs/jwt-ory-to-istio.md

```diff
@@ -0,0 +1,131 @@
+---
+title: JWT access strategies with Ory Oathkeeper to Istio
+---
+
+We are in the process of transitioning from Ory Oathkeeper to Istio JWT access strategy. This document explains the differences between those two strategies and compares their configuration.
+
+## Comparison of Ory Oathkeeper and Istio JWT access strategy configurations
+
+>**CAUTION:** Istio JWT is **not** a production-ready feature, and API might change.
+
+These are sample APIRule custom resources of both Ory Oathkeeper and Istio JWT access strategy configuration for a service.
+
+<div tabs name="api-rule">
+  <details>
+  <summary>
+  Ory Oathkeeper
+  </summary>
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta1
+kind: APIRule
+metadata:
+  name: service-secured
+spec:
+  gateway: kyma-system/kyma-gateway
+  host: foo.bar
+  service:
+    name: foo-service
+    namespace: foo-namespace
+    port: 8080
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      mutators: []
+      accessStrategies:
+        - handler: jwt
+          config:
+            trusted_issuers:
+              - $ISSUER1
+              - $ISSUER2
+            jwks_urls:
+              - $JWKS_URI1
+              - $JWKS_URI2
+```
+
+  </details>
+  <details>
+  <summary>
+  Istio
+  </summary>
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta1
+kind: APIRule
+metadata:
+  name: service-secured
+  namespace: $NAMESPACE
+spec:
+  gateway: kyma-system/kyma-gateway
+  host: foo.bar
+  service:
+    name: foo-service
+    namespace: foo-namespace
+    port: 8080
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      mutators: []
+      accessStrategies:
+        - handler: jwt
+          config:
+            authentications:
+            - issuer: $ISSUER
```

Comment:
In the Oathkeeper example we used `$ISSUER1`, maybe we should use the same name here.
```suggestion
            - issuer: $ISSUER1
```